robotbase/src/main/java/frc/robot/Constants.java

```diff
@@ -162,6 +162,9 @@ public static final class SHOOTER {
   }
 
   public static final class INTAKE {
+    public static final double TOP_MOTOR_SPEED_PERCENT_OUTPUT = 0;
```

Comment:
We will likely need to separate speeds from intaking the note and feeding the shooter. There should be a pair of constants, one for the `IntakeNoteCommand` and one for the `IntakeToShooterCommand`